### ISSUE TITLE
Omen of clarity

### ISSUE THREAD

**Issue by blzee** [OmegaWoWeu/Decades#118]
Description: 
http://wotlk.cavernoftime.com/spell=48467
http://wotlk.cavernoftime.com/spell=16870

Clearcasting has chance to proc per target hit by Hurricane. 

Rather than proccing per target hit, it should only have a chance to proc per tick of the channel.

Unlike the Mage talent Arcane Concentration, a druid's clearcasting should have a chance to proc per tick of a channeled ability.

A source happens to mention this: http://www.clanyawa.com/viewtopic.php?t=5411
"Omen of Clarity - Gives a chance on melee autoattack and spellcast to cause your next damage or healing spell cast within 12 seconds to have no mana cost. This is considered a standard talent for Balance. The actual proc rate is supposed to average 6 per minute. Interestingly, it has a chance to proc for each tick of a channeled spell like Hurricane."

How to reproduce bug: make a druid and test, but I’ve tested it pretty thoroughly.

How it should work: The proc rate is about 6%. Everything except shapeshifting can cause proc and not all spells will benefit from it. Only healing spells/dmg spells benefit from proc.

TL:DR Druids are not getting our clear casting procs from hurricane.

Screenshots: can’t really SS it, but I provided logs.

Relevant AddOns used: n/a

Source:
http://www.worldoflogs.com/reports/ak6ponc2cou5kufz/xe/?x=spell+%3D+%22Gift+of+the+Wild%22%0D%0AAND+sourceName+%3D+%22Oranoss%22%0D%0AOR+spell+%3D+%22Clearcasting%22%0D%0AAND+sourceName+%3D+%22Oranoss%22

**Comment by ghost** [comment]
My boomkin has never gotten a clearcasting proc from Hurricane. From your report it sounds like they're getting _too many_ procs.

**Comment by blzee** [comment]
No, I think you’re misinterpreting it. I’m agreeing with you 100%. We aren’t getting our clear casting procs from hurricane like we are supposed to.

**Comment by ghost** [comment]
Oh "Rather than proccing per target hit, it should only have a chance to proc per tick of the channel." sounded like we were getting procs depending on the amount of targets hit with hurricane, which would be a lot. I understand now.

**Comment by Rtw0** [comment]
yes I noticed this beheavior too, only from initial cast but not from channeled ticks.

**Comment by Rtw0** [comment]
I just want to add that this is not just Hurricane ticks but all the DOTS from Balance druid do not proc any Clearcasting, not even Starfall does proc any clearcasting. Currently Clearcasting can proc only on Initial Cast which is not intended game mechanic.
OoC tooltip for reference:

> Each of the Druid's damage, healing spells and auto attacks has a chance of causing the caster to enter a Clearcasting state. The Clearcasting state reduces the Mana, Rage or Energy cost of your next damage, healing spell or offensive ability by 100%.
